### ISSUE TITLE
Search parameters need changing

### ISSUE THREAD

**Issue by ChasCB** [johannhof/emoji-helper#116]
If you, for example, search for clown face, you will not find it. But if you search for clown_face you find it. Another example is new_zealand or south_africa.

For beginners this would be very frustrating not knowing you have to put in the underscore line between each word. Is there some way you can code for the underscore without having to put it into the search term or even remove the underscore entirely from the emoji's descriptions.

Also the search term must be all lower case. But the natural thing to do for countries is Capitalise. e.g. New Zealand, South Africa.

Cheers

**Comment by patricktokeeffe** [comment]
Yes, case-sensitive search produces hassle for cut-and-paste terms or things you expect to be capitalized
